docs/data-sources/scm_certificates.md

```diff
@@ -43,17 +43,19 @@ The options are as follows:
 
 * `deploy_support` - (Optional, Bool) Whether to support deployment.
 
+* `name` - (Optional, String) Certificate name.
```

Comment:
if `name` is used more commonly, suggest to put it in the front

Reply:
ok,done